packages/core/src/tracing/backend_mappers/mapper.js

```diff
@@ -31,6 +31,11 @@ const fieldMappings = {
   kafka: {
     operation: 'access',
     endpoints: 'service'
+  },
+  http: {
```

Comment:
suggestion: The fieldMappings object is only for fields who **need** a mapping. Fields like `redis.connection` are already normalized and are not listed here. To avoid confusion, could you please add these fields as a comment?

```
redis: {
  operation: 'command'
  // connection: connection # already normalized
}